gestalt/__init__.py

```diff
@@ -611,11 +596,60 @@ def fetch_vault_secrets(self) -> None:
             try:
                 secret_token = self.vault_client.secrets.kv.v2.read_secret_version(
                     mount_point=str(mount_path), path=secret_path)
+                print(secret_token)
                 self.__conf_data.update(secret_token['data']['data'])
+                if secret_token['lease_id'] != '': 
+                    secret_lease = (secret_token['lease_id'],
```

Comment:
I feel like this should be it's own class in place of a tuple, for type safety.

Reply:
Can now make it its own data class since it is evolving rapidly